lib/transport_web/controllers/session_controller.ex

```diff
@@ -15,19 +15,21 @@ defmodule TransportWeb.SessionController do
   end
 
   def create(conn, %{"code" => code}) do
-    client = get_token!(code)
-    user   = get_user!(client)
-
-    conn
-    |> put_session(:current_user, user)
-    |> put_session(:access_token, client.token.access_token)
-    |> redirect(to: "/search_organizations")
+    with client <- get_token!(code),
+         user <- get_user!(client) do
+      conn
+      |> put_session(:current_user, user)
+      |> put_session(:access_token, client.token.access_token)
+      |> redirect(to: user_path(conn, :organizations))
+      |> halt()
```

Comment:
You're not in a plug, so you don't need a `halt`

Reply:
No.